pkg/cluster/internal/providers/docker/network.go

```diff
@@ -275,8 +275,12 @@ func isNetworkAlreadyExistsError(err error) bool {
 }
 
 // returns true if:
+// - err is nil
```

Comment:
why? this seems wrong? it can't be a particular kind of error if it's nil / not even an error